src/server/services/get-template.js

```diff
@@ -28,7 +23,7 @@ const validation = {
 
 module.exports = {
   getDropdownValues,
-  getTreasuryTemplateSheets,
+  getTemplate,
```

Comment:
nit: i would have named `getTemplate` as `loadTemplateCached` and then exported `loadTemplate: loadTemplateCached` to make it clearer what's happening, this would have prevented changes downstream as well

Reply:
I generally like verbose names too.  I just went with getTemplate, because I noticed that all of the functions in this file use an existing pattern:

- `loadX` actually calls the filesystem
- `getX` wraps `loadX` with caching logic

I don't really want to rename everything in this change, but I'm happy to do so afterward!